packages/haste-task-typescript/test/index.spec.js

```diff
@@ -0,0 +1,96 @@
+const fs = require('fs-extra');
+const path = require('path');
+const tempy = require('tempy');
+const { run } = require('haste-test-utils');
+
+const { command: typescript, kill } = run(require.resolve('../src'));
+const configPath = require.resolve('./fixtures/tsconfig.json');
+const transpiledFixture = fs.readFileSync(require.resolve('./expected/valid.transpiled'), 'utf-8');
+const sourcemapFixture = fs.readFileSync(require.resolve('./expected/valid.js.map'), 'utf-8');
+
+jest.setTimeout(30000);
+
+describe('haste-task-typescript', () => {
+  afterEach(kill);
+  let projectDir;
+  let outDir;
+
+  beforeEach(() => {
+    projectDir = tempy.directory();
+    outDir = path.join(projectDir, 'dist');
+    fs.copySync(configPath, path.join(projectDir, 'tsconfig.json'));
+  });
+
+  it('should transpile with typescript and resolve', async () => {
+    fs.copySync(require.resolve('./fixtures/valid.ts'), path.join(projectDir, 'src/valid.ts'));
```

Comment:
I think it would be easier to understand if we copy all needed files at the same time instead of copying the config once in a `beforeEach` hook and then other files inside the test, how about something like:
```js
copy({
  filename: 'fileconent',
  ...
});
```